src/services/pegout-status/pegout-status.service.ts

```diff
@@ -39,14 +37,17 @@ export class PegoutStatusService {
                 .then(async (pegoutStatusDbDataModel) => {
                     if (!pegoutStatusDbDataModel) {
 
-                        // TODO implements the new getTransactionMethod in rskNodeService
-                        const transaction:RskTransaction = await this.rskNodeService.getTransaction(rskTxHash);
-                        // TODO implements the receipt validation 
-                        //const receipt = await this.rskNodeService.getTransactionReceipt(rskTxHash);
+                        const rskTransaction:RskTransaction = await this.rskNodeService.getTransaction(rskTxHash, this.ATTACH_TRANSACTION_RECEIPT);
                         
-                        if(transaction) {
-                            //Process the transaction using the same rules in pegout-data-processor
-                           //pegoutStatus = await this.processTransaction(transaction);
+                        if(rskTransaction) {
+                            const receipt = rskTransaction.receipt;
+                            if(receipt) {
+                                const transaction:Transaction = await this.rskNodeService.getBridgeTransaction(rskTxHash);
```

Comment:
why do we need to get the transaction twice? do we truly need the receipt? the Daemon is doing all the work just with the BridgeTransaction, can't we do the same?